src/app/static/src/app/index.ts

```diff
@@ -1,19 +1,36 @@
 import Vue from "vue";
 import {store} from "./main"
 import Stepper from "./components/Stepper.vue";
+import ForgotPassword from "./components/password/ForgotPassword.vue";
 import {mapActions} from "vuex";
 
-export const app = new Vue({
-    el: "#app",
-    store,
-    components: {
-        Stepper
-    },
-    render: h => h(Stepper),
-    methods: {
-        ...mapActions({loadBaseline: 'baseline/getBaselineData'})
-    },
-    beforeMount: function() {
-        this.loadBaseline()
-    }
-});
+export const app =
```

Comment:
So this might be completely the wrong approach! I wanted to do the ForgotPassword component as vue rather than a server template because I thought that would be nicer to show response messages in js than just using an html form. But then it didn't seem quite right to use the standard 'app' vue element that you get once you're logged, so I've done this, choosing the component to render based on element found in the document (as defined by the template). It does still use the same store type as before, but with a password module added. 

An alternative would be to have this render a single root component which conditionally renders a Stepper or a ForgotPassword - not sure what the most correct basis for this decision would be though..

Reply:
I'm not sure what advantage this exact approach offers because the same js bundle would have to be included on both the ForgottenPassword page and the regular app page. 

But I can see why splitting up the 2 into separate apps makes sense i.e. two separate entrypoints for webpack. So the forgotten password vue app would live in a different file and be bundled into it's own bundle, then each page only has to load the js it needs.